code/modules/mining/mine_items.dm

```diff
@@ -579,13 +579,12 @@ proc/move_mining_shuttle()
 		return
 	..()
 
-/mob/living/simple_animal/hostile/mining_drone/Die()
-	..()
+/mob/living/simple_animal/hostile/mining_drone/death(var/gibbed=FALSE)
+	..(TRUE)
```

Comment:
Because I generally consider the gibbed argument as being "no body", and since these things explode on death we just ignore whether it was gibbed or not.